Chapter09/recipe-05/cxx-example/account/CMakeLists.txt

```diff
@@ -15,16 +15,19 @@ add_library(
   account
   MODULE
   account.cpp)
+
 target_link_libraries(
   account
   PRIVATE
   pybind11::module
   )
+
 target_include_directories(
   account
   PRIVATE
   ${PYBIND11_DIR}/include
```

Comment:
I don't think it's necessary to also use `target_include_directories` here. Linking to the imported target will fix that for you.

Reply:
You are right - thank you. I will remove this.